docs/book/modules/concepts/pages/index.adoc

```diff
@@ -0,0 +1,88 @@
+= Concepts
+
+The following sections explain a few concepts of Trustify.
+
+[#vulnerability]
+== Vulnerability
+
+A vulnerability is mostly, primarily a *name* that is used to ensure all advisories are discussing the same thing.
+Generally, to this point, most vulnerabilities come from the CVE Project, with the format of `CVE-2024-1234`.
+
+Within the database, generally a vulnerability is added as a side effect of an advisory mentioning it.
+
+A *CVE Record* from NIST/NVD is a low-value advisory that is generally the first discovered advisory that mentions a vulnerability.
+
+== Advisory
+
+An advisory is an opinion about a vulnerability.
+
+These opinions include the context to which the opinions apply.
+These opinions include evaluation of the severity and scoring of a vulnerability within that context, such as CVSS scores.
+
+As mentioned above, a *CVE Record* from the CVE Project is a low-value advisory that mentions the vulnerability and provides a base opinion about it.
+It may include CVSS scores, within the context of the abstract origin containing the vulnerability.
+This may be simply in reference to the vulnerability *as it exists in source-code form*.
+
+Other, more-involved stakeholders (product vendors, upstream project owners) may issue *additional* advisories.
+These opinions may be in reference to *concrete* shipped products, contextualized to how the vulnerable code is *actually used*.
+
+== SBOM
+
+An SBOM is a source-of-someone's-truth about "what's inside it?", so
+everything in our DB is ultimately sourced from some
+source-of-truth. We can't really say definitively "product X is
```

Comment:
We should not use contractions in technical documentation.

`can't` --> `can not`